src/Files.App/ServicesImplementation/Settings/FoldersSettingsService.cs

```diff
@@ -207,6 +207,25 @@ public bool CalculateFolderSizes
 			get => Get(false);
 			set => Set(value);
 		}
+
+		public SortDirection DefaultDirectorySortDirection
```

Comment:
We don't plan on this option for now

Reply:
Ok, I will remove it

Reply:
It should be kept in LayoutSettingsService but not here as we don't want to expose the setting.